### PR TITLE
add  turbosms(UA) sms  provider

### PR DESCRIPTION
**Pull request by dzirg44**
I added  sms  provider from Ukraine to sachet (https://turbosms.ua/) . My code is very ugly, but  it work (i am not programmer, but learning golang and use prometheus). I will be very grateful if someone refacts this code.

### CODE REVIEW

**Comment by samwierema** [comment]
Hey @dzirg44! Thanks for opening a pull request! Could you remove the `sachet.service` file? This is a specific implementation for you and is beyond the scope of this repository.

**Comment by dzirg44** [comment]
I thought the examples directory might contain this file. I deleted this file, and I expect to merge.

File: turbosms.go [hunk 1]
```diff
@@ -0,0 +1,142 @@
...
+	clientConfig := &http.Client{
+		Timeout: 15 * time.Second,
+		Jar:     cookieJar,
+	}
+	////// Request ///////////
+	reply, err, statusreply := Request(clientConfig, urlSoap, []byte(data))
+	if err != nil {
+		return err
+	}
+	replysms, err, statusreplysms := Request(clientConfig, urlSoap, []byte(datasms))
```
Comment (marcelcorso) [inline turbosms.go]: You should only do the SMS request if the Auth request succeded.

Comment (dzirg44) [inline turbosms.go]: i know but , response from provider may  be changed. I implemented this in the early version (the decoding function), but I did not get a reliable delivery mechanism. I can return this functionality if you find it necessary.

Comment (marcelcorso) [inline turbosms.go]: I don't understand what you mean with "response from provider may be changed". 
:-)

Comment (dzirg44) [inline turbosms.go]: At authorization on request -
`<?xml version="1.0" encoding="UTF-8"?>
<SOAP-ENV:Envelope xmlns:SOAP-ENV="http://schemas.xmlsoap.org/soap/envelope/" xmlns:ns1="http://turbosms.in.ua/api/Turbo">
  <SOAP-ENV:Body>
    <ns1:Auth>
      <ns1:login>login</ns1:login>
      <ns1:password>212332</ns1:password>
    </ns1:Auth>
  </SOAP-ENV:Body>
</SOAP-ENV:Envelope>`

Comes the right answer -
`<?xml version="1.0" encoding="utf-8"?>
<SOAP-ENV:Envelope xmlns:SOAP-ENV="http://schemas.xmlsoap.org/soap/envelope/" xmlns:ns1="http://turbosms.in.ua/api/Turbo">
        <SOAP-ENV:Body>
                <ns1:AuthResponse>
                        <ns1:AuthResult>Вы успешно авторизировались</ns1:AuthResult>
                </ns1:AuthResponse>
        </SOAP-ENV:Body>
</SOAP-ENV:Envelope>`
But I'm not sure that the answer will always be like that (Вы успешно авторизировались). 
And in the php class that the developer provides (https://turbosms.ua/soap.html), I did not find such a check, so I thought that the server's response might change.

**Comment by dzirg44** [comment]
Sending SMS to several recipients does not work, because  slice has spaces . I used the package string to fix the problem.

**Comment by samwierema** [comment]
Hey @dzirg44, everything is good now (including sending to multiple recipients)?

**Comment by dzirg44** [comment]
everything is ok now. i tested  in production  - sms  sending without any  problems  for  multiple  recipients .

**Pull request merged**